src/pog/anv.cpp

```diff
@@ -23,9 +23,13 @@ namespace pog
         return db.GetAllANVs();
     }
 
-    referral::AddressANVs GetAllRewardableANVs(const referral::ReferralsViewDB& db)
+    void GetAllRewardableANVs(
```

Comment:
Can we name this function differently? maybe LoadAllRewardableANVs? 
It's quite confusing that we update `entrants` and don't return anything right now.